src/components/tonies/TonieCardDisplayOnly.tsx

```diff
@@ -62,7 +62,7 @@ export const TonieCard: React.FC<{ tonieCard: TonieCardProps }> = ({ tonieCard }
     );
     const title = `${tonieCard.tonieInfo.series} - ${tonieCard.tonieInfo.episode}`;
     const more = [
-        isValid ?
+        tonieCard.valid ?
```

Comment:
Are you sure, this is wrong? If you don't use isValid, the variable will only be updated if you reload the page.

Reply:
As alternative, if there are no changes made to it, remove isValid completely.

Reply:
On this kind of card is no download of the taf planned. So it stays unchanged. I would leave it as it is, since it works correct so. If there is no file to play available, the valid flag is false, so the play icon is grayed out, is a file available, flag is true and everything is fine.

Reply:
Using isValid before disables some tonies where it shouldn't be disabled. After changing it, it fits the real situation.

Reply:
so i removed the const as recommended